openscap/xml/article_openscap.xml

```diff
@@ -547,13 +624,21 @@
       <para>
         Automated checks help to identify the target and to select only the
         rules that make sense for this specific target. For example, checks for
-        separate partitions make sense for bare-metal machines but not for
+        separate partitions make sense for bare metal machines but not for
```

Comment:
The suse-vale extension for VS Code suggests using "bare metal" instead of "bare-metal". But I am going according to your suggestion. If the rule in the extension is outdated, we need to file a bug/issue.

Reply:
Yes, unfortunately the extension makes no difference between the noun and the adjective. We got it in our style guide: noun = bare metal, adj. = bare-metal